[ADD] VoIP: Axivox Documentation Grouping (#5256)
odoo/documentation: content/applications/general/voip/axivox/vm_audio_messages.rst

=============================

.. _axivox_admin: https://manage.axivox.com/

Managing voicemail is an important part of any business. A company needs to access their messages
with ease and stay on top of any missed calls. Recording audio messages, like thanking a caller for
reaching out, or directing them to the right extension, is also a great way to personalize the
business interaction, and set the tone with the customer.

This document covers the configuration of both voicemail and audio message in the Axivox
administrative portal.

.. _axivox/global_language:

Set global language
===================

To start the *global language* should be set in the Axivox admin portal settings. Navigate to
`manage.axivox.com <axivox_admin_>`_. After logging into the portal go to :menuselection:`Settings
--> Global language (e.g.: voicemail messages,...)`. Set the language to
either: :guilabel:`Francais`, :guilabel:`English`, :guilabel:`Espanol`, or :guilabel:`Deutsch`.

Click :guilabel:`Save` and then :guilabel:`Apply changes` in the upper right of the screen to
implement the change into production.

.. _axivox/activate_voicemail:

Activate voicemail
==================

> In order for a user to utilize voicemail in Axivox the voicemail feature needs to be turned on in
> the Axivox administrative portal. To begin using voicemail with a user navigate to
> `manage.axivox.com <axivox_admin_>`_. Log in with the appropriate administrator credentials. On the
> left menu of the Axivox administrative panel, click into :guilabel:`Users`.
> 
> Click into the specific user that the voicemail should be activated in. Under the section marked
> :guilabel:`Voicemail`, select the drop-down menu and click on :guilabel:`Yes`. :guilabel:`Save` the
> change and :guilabel:`Apply changes` in the upper right of the screen.

Reviewer: ```suggestion
In order for a user to utilize voicemail in Axivox, the voicemail feature **must** be turned on in
the Axivox administrative portal. To begin using voicemail with a user, navigate to
`manage.axivox.com <axivox_admin_>`_. Then, log in with the appropriate administrator credentials.

On the left menu of the Axivox administrative panel, click into :guilabel:`Users`.

Then, click into the specific user the voicemail should be activated for. Under the section marked,
:guilabel:`Voicemail`, open the drop-down menu, and click on :guilabel:`Yes`.

Lastly, :guilabel:`Save` the change, then click :guilabel:`Apply changes` in the upper-right corner
of the screen.
```

Voicemail
=========

> The next step is to set up the individual voicemail boxes on the Axivox administrative portal. To
> access the portal visit `manage.axivox.com <axivox_admin_>`_ and login. Navigate to
> :guilabel:`Voicemails` in the left menu. If the voicemail option was activated in the user profile
> using this process :ref:`axivox/activate_voicemail`, then a voicemail is automatically created in
> this :menuselection:`Voicemails` section.
> 
> .. tip::
>    It should be noted that some of the administrative portal language will be in French as Axivox is
>    a Belgian company. The global language is still set to one of the four options as seen here:
>    :ref:`axivox/global_language`
> 
> Manually create voicemail
> -------------------------
> 
> To manually create a new voicemail box, click on :guilabel:`Add a voicemail` or to edit an existing
> voicemail box click :guilabel:`Edit` to the far right of the existing voicemail box.
> 
> .. example::
>    Suppose a sales or support team needs a general voicemail box. The voicemail would need to be
>    created manually and attached to an incoming number.
> 
> The newly manually created voicemail box should be attached to an incoming number so that it can
> receive messages. To do so, first, navigate to :guilabel:`Incoming numbers` in the left menu. Then
> click :guilabel:`Edit` to the far right of the specific number that the voicemail should be linked
> to.
> 
> Under :guilabel:`Destination type for voice call` click the drop-down menu and select
> :guilabel:`Voicemail`. Then select the drop-down menu on the next line labeled
> :guilabel:`Voicemail`, and select the manually created voicemail box. Enter a :guilabel:`Destination
> email address for Incoming SMS` and click :guilabel:`Save`. :guilabel:`Apply changes` in the upper
> right side of the screen to implement the change into production.
> 
> .. tip::
>    Under the field labeled: :guilabel:`Destination email address for Incoming SMS` enter an email
>    to which incoming text messages sent to the incoming number can be received. Some incoming
>    numbers (US +1) in Axivox are capable of receiving text messages from individuals and automated
>    numbers. Should this field be left empty then the default destination address is used instead
>    (as previously set here in the start of manually creating a voicemail). To determine whether an
>    incoming number is capable of receiving SMS/text messages click on :guilabel:`Incoming numbers`
>    in the left menu. Then check the :guilabel:`SMS compatible` column for the incoming number.
> 
> Notifications
> -------------
> 
> Anytime a voicemail is received now on any of the automatically pre-configured or manually linked
> voicemail boxes, an email is sent to the user's email address as listed in the
> :menuselection:`Voicemails` screen or in the user's Axivox profile. This can be accessed by
> navigating to :guilabel:`Users` in the left menu and clicking on :guilabel:`Edit` next to the
> specific user in question.

Reviewer: ```suggestion
The next step is to set up the individual voicemail boxes on the Axivox administrative portal. To
access the portal, visit `manage.axivox.com <axivox_admin_>`_ and log in. Then, navigate to
:menuselection:`Voicemails`, located in the menu on the left.

If the voicemail option was activated in the user profile, using this process
:ref:`axivox/activate_voicemail`, then a voicemail is automatically created on the
:guilabel:`Voicemails` pages.

.. tip::
   It should be noted that some of the administrative portal language is in French, as Axivox is a
   Belgian company. The global language is still set to one of the four options as seen here:
   :ref:`axivox/global_language`

Manually create voicemail
-------------------------

To manually create a new voicemail box, click :guilabel:`Add a voicemail` on the
:guilabel:`Voicemails` page. Or, edit an existing voicemail box, by clicking :guilabel:`Edit` to the
far-right of an existing voicemail box on the :guilabel:`Voicemails` page.

.. example::
   Suppose a sales or support team needs a general voicemail box. The voicemail would need to be
   created manually, and attached to an incoming number.

The new, manually-created voicemail box should be attached to an incoming number, so it can receive
messages. To do so, navigate to :menuselection:`Incoming numbers`, located in the menu on the left.
Then, click :guilabel:`Edit` to the far-right of the specific number the voicemail should be linked
to.

In the :guilabel:`Destination type for voice call` field, click the drop-down menu, and select
:guilabel:`Voicemail`. Then, open the drop-down menu on the next line labeled,
:guilabel:`Voicemail`, and select the manually-created voicemail box.

.. important::
   If an incoming number is capabable of receiving SMS/text messages, an additional field,
   :guilabel:`Destination email address for Incoming SMS` is present.

   To determine whether an incoming number is capable of receiving SMS/text messages, click
   :guilabel:`Incoming numbers` from the menu on the left, then check the :guilabel:`SMS compatible`
   column for the incoming number.

Then, if applicable, in the field labeled, :guilabel:`Destination email address for Incoming SMS`,
enter an email to which incoming text messages sent to the incoming number can be received. Some
incoming numbers (US +1) in Axivox are capable of receiving text messages from individuals and
automated numbers.
   
Should this field be left empty, the default destination address is used, instead (as previously set
in the beginning of the process for manually creating a voicemail).

Once all desired configurations are complete, click :guilabel:`Save`, then click :guilabel:`Apply
changes` in the upper-right corner of the screen to implement the change into production.

Notifications
-------------

Now, whenever a voicemail is received on any of the automatically pre-configured, or
manually-linked, voicemail boxes, an email is sent to the user's email address, as listed in the
:guilabel:`Voicemails` page, or in the user's Axivox profile.

This information can be accessed by navigating to :menuselection:`Users` in the left menu, and
clicking :guilabel:`Edit` next to the specific user in question.
```

a :guilabel:`Name` and click :guilabel:`Save`.

Upon clicking :guilabel:`Save`, the browser redirects back to the main :guilabel:`Audio messages`
page. There are two different ways to actually make the audio message, either by recording the
message over the phone or by typing in text and selecting a computer generated speaker to read the
message.

Record audio message
--------------------

The first option available is to record a message over the phone by clicking on the orange button
labeled: :guilabel:`Record/Listen`. The message is recorded via one of the extensions that is
associated with a user.

Under :guilabel:`Extension to user for message management` click the drop-down menu and select the
extension where Axivox should call to record the message. Then click :guilabel:`OK` to begin the
call.

.. note::
   The user must be active in the production database with :abbr:`VoIP (Voice over Internet
   Protocol)` configured. To configure :abbr:`VoIP (Voice over Internet Protocol)` for a user see
   this documentation: :doc:`axivox_config`.

Upon connecting to the Axivox audio recorder management line a recorded French speaking operator
provides the following options:

> #. Press `1` to record a message.
> #. Press `2` to listen to the current message.
> 
> Press either `1` or `2` depending on whether there is already a message present in the system for
> this particular audio message (press `2`) or if a new audio message needs to be recorded (press
> `1`).
> 
> Record the audio message after pressing `1`, then press `#` to end the recording. The French
> speaking operator returns to the line presenting the first set of questions again:
> 
> #. Press `1` to record a message.
> #. Press `2` to listen to the current message.
> 
> Press `#` to end the call.
> 
> Write audio message
> -------------------
> 
> The second option to obtain an audio message is to type the message out and select a computerized
> speaker to say the text. To do so, simply, navigate to the :guilabel:`Audio messages` page by
> clicking on :guilabel:`Audio messages` in the left panel. Then select the blue button labeled:
> :guilabel:`Text message`, next to the corresponding audio message :guilabel:`Name` that the audio
> message should be attached to.
> 
> Click the drop-down menu next to the field labeled :guilabel:`Voice` and select a :guilabel:`Voice` for
> the :guilabel:`Text` to be read in. After the selection has been made, click :guilabel:`Generate` to
> process the audio file. The text is read in the language that it is written in the :guilabel:`Text`
> field. Should the language differ in the :guilabel:`Voice` then an accent is used by the
> computerized speaker.
> 
> 
> Finally when these steps are complete, click :guilabel:`Save` to save the audio message.
> 
> To implement the changes, click :guilabel:`Apply changes` in the upper right-side of the screen.
> 
> .. tip::
>    To use a audio message associated with a dial plan menu element see this documentation
>    :doc:`dial_plan_basics` or :doc:`dial_plan_advanced`.
> 
> .. _axivox/music_on_hold:
> 
> Music on-hold
> =============
> 
> Axivox has the option to add custom hold music to the call whenever a caller is waiting for their
> call to be answered. To add *hold music* on to the Axivox administrative portal, navigate to the
> `manage.axivox.com <axivox_admin_>`_, and login. Click on :guilabel:`Music on hold` and a pop-up
> appears.
> 
> On the :menuselection:`Change the music on hold` pop-up, click on the :guilabel:`Browse...` button
> to select the MP3 (MPEG Audio Layer 3) or WAV (Waveform Audio File Format ) file to be uploaded.
> 
> .. note::
>    Only :abbr:`MP3 (MPEG Audio Layer 3)` or :abbr:`WAV (Waveform Audio File Format)` files can be
>    uploaded to the Axivox administrative portal.
> 
> Once the file is selected, the :guilabel:`Progression` bar shows an upload status. When this
> activity completes, then the window can be closed, by clicking :guilabel:`Close`.
> 
> :guilabel:`Apply changes` in the upper right-side of the screen.

Reviewer: ```suggestion
In Axivox, there are also numerous forwarding settings for a user. To access these forwardings
settings, go to `manage.axivox.com <axivox_admin_>`_ and log in.

Next, navigate to :menuselection:`Users`, located in the menu on the left.

From there, click into the specific user the forwarding should be added to. Then, open the
:guilabel:`Forwardings` tab.

If the user is busy on another call, or away from the phone, there is an option present in this tab
to :guilabel:`Send to voicemail as a last resort`, located in the :guilabel:`Forwarding on no
answer` and :guilabel:`Forwarding on busy` fields.

.. image:: vm_audio_messages/forwardings.png
   :align: center
   :alt: Send to voicemail as a last resort options highlighted on the Forwardings tab of the user.

If the :guilabel:`Send to voicemail as a last resort` box is ticked, when the forwarding actions
stated in each section are not successful, the caller is routed to the voicemail set on the
particular user.

.. seealso::
   For more information on forwarding and transfers visit :ref:`axivox/forwardings_tab`.

When all the desired configurations are complete, click :guilabel:`Save`, then click
:guilabel:`Apply changes` in the upper-right corner of the screen to implement the change.

Audio messages
==============

It is possible to add audio messages *before* a customer's call is even taken, to inform them about
the waiting time for deliveries, the availability of a product, or any other important promotional
messages.

To record an audio message in Axivox, navigate to `manage.axivox.com <axivox_admin_>`_ and log in.

Next, click on :guilabel:`Audio messages` in the menu on the left. From the :guilabel:`Audio
messages` page, click :guilabel:`Add a message`.

Type in a :guilabel:`Name`, and click :guilabel:`Save`.

Upon clicking :guilabel:`Save`, the browser redirects back to the main :guilabel:`Audio messages`
page, where the newly-created message can be found on the list.

There are two different ways to make the audio message. The user could either record the message
over the phone, or type the message (in text), and select a computer-generated speaker to read the
message.

Record audio message
--------------------

To record an audio message over the phone, click the orange button labeled,
:guilabel:`Record/Listen`, located to the right of the desired message on the list to record for, on
the :guilabel:`Audio messages` page.

When clicked, a :guilabel:`Record / listen to a message` pop-up window appears. From here, the
message is then recorded, via one of the extensions that is associated with the user.

Under :guilabel:`Extension to use for message management` field, click the drop-down menu, and
select the extension where Axivox should call to record the message.

Then, click :guilabel:`OK` to begin the call.

.. note::
   The user **must** be active in the production database with :abbr:`VoIP (Voice over Internet
   Protocol)` configured. To configure :abbr:`VoIP (Voice over Internet Protocol)` for a user, see
   this documentation: :doc:`axivox_config`.

Upon connecting to the Axivox audio recorder management line, a recorded French-speaking operator
provides the following options:

#. Press `1` to record a message.
#. Press `2` to listen to the current message.

Press either `1` or `2`, depending on whether or not there is already a message present in the
system for this particular audio message that requires a review, before recording a new one.

Record the new audio message after pressing `1`, then press `#` to end the recording.

The French-speaking operator returns to the line presenting the first set of questions again:

#. Press `1` to record a message.
#. Press `2` to listen to the current message.

Press `#` to end the call.

Write audio message
-------------------

To type the message, and select a computerized speaker to say the text, navigate to the
:menuselection:`Audio messages` in the menu on the left.

From the :guilabel:`Audio messages` page, select the blue button labeled, :guilabel:`Text message`,
next to the corresponding audio message :guilabel:`Name` that the message should be attached to.

Doing so reveals :guilabel:`Convert text to message` pop-up window.

From the :guilabel:`Convert to text message` pop-up window, click the drop-down menu next to the
field labeled, :guilabel:`Voice`, and select an option for the :guilabel:`Text` to be read in.

After the :guilabel:`Voice` selection has been made, and the message has been written in the
:guilabel:`Text` field, click :guilabel:`Generate` to process the audio file.

The text is read in the same language it is written in the :guilabel:`Text` field. Should the
language differ in the :guilabel:`Voice`, then an accent is used by the computerized speaker.

Finally, when these steps are complete, click :guilabel:`Save` to save the audio message.

To implement the changes, click :guilabel:`Apply changes` in the upper-right corner of the screen.

.. tip::
   To use an audio message associated with a dial plan menu element, see this documentation
   :doc:`dial_plan_basics` or :doc:`dial_plan_advanced`.

.. _axivox/music_on_hold:

Music on-hold
=============

Axivox has the option to add custom hold music to the call whenever a caller is waiting for their
call to be answered. To add hold music on to the Axivox administrative portal, navigate to the
`manage.axivox.com <axivox_admin_>`_, and log in.

Then, click on :guilabel:`Music on hold` from the menu on the left, and a :guilabel:`Change the
music on hold` pop-up window appears.

On the :menuselection:`Change the music on hold` pop-up window, click the :guilabel:`Choose File`
button to select an MP3 (MPEG Audio Layer 3) or WAV (Waveform Audio File Format ) file to be
uploaded.

.. note::
   Only :abbr:`MP3 (MPEG Audio Layer 3)` or :abbr:`WAV (Waveform Audio File Format)` files can be
   uploaded to the Axivox administrative portal.

Once the file is selected, the :guilabel:`Progression` bar shows an upload status. When this
activity completes, the window can be closed, by clicking :guilabel:`Close`.

When the desired changes are complete, click :guilabel:`Apply changes` in the upper-right corner of
the screen.
```
this doc might be even more beneficial with a few more screenshots, but that's just a suggestion

To implement the changes, click :guilabel:`Apply changes` in the upper right-side of the screen.

.. tip::
   To use a audio message associated with a dial plan menu element see this documentation
   :doc:`dial_plan_basics` or :doc:`dial_plan_advanced`.

.. _axivox/music_on_hold:

Music on-hold
=============

Axivox has the option to add custom hold music to the call whenever a caller is waiting for their
call to be answered. To add *hold music* on to the Axivox administrative portal, navigate to the
`manage.axivox.com <axivox_admin_>`_, and login. Click on :guilabel:`Music on hold` and a pop-up
appears.

On the :menuselection:`Change the music on hold` pop-up, click on the :guilabel:`Browse...` button
to select the MP3 (MPEG Audio Layer 3) or WAV (Waveform Audio File Format ) file to be uploaded.

.. note::
   Only :abbr:`MP3 (MPEG Audio Layer 3)` or :abbr:`WAV (Waveform Audio File Format)` files can be
   uploaded to the Axivox administrative portal.

Once the file is selected, the :guilabel:`Progression` bar shows an upload status. When this
activity completes, then the window can be closed, by clicking :guilabel:`Close`.

> :guilabel:`Apply changes` in the upper right-side of the screen.

Author: Duplicate from last revision? @ksc-odoo

> :guilabel:`Apply changes` in the upper right-side of the screen.

Reviewer: Seems like a duplicate, indeed. Strange how that happened. Yes, if you've already made this necessary change(s), you can omit this duplicate, and continue onward with your revisions. Thanks!

=============================
Voicemails and audio messages
=============================

.. _axivox_admin: https://manage.axivox.com/

Managing voicemail is an important part of any business. A company needs to access their messages
with ease, and stay on top of any missed calls. Recording audio messages, like thanking a caller for
reaching out, or directing them to the right extension, is also a great way to personalize the
business interaction, and set the tone with the customer.

> This document covers the configuration of both voicemail and audio message in the Axivox

Reviewer: ```suggestion
This document covers the configuration of both voicemail and audio messages in the Axivox
```

.. _axivox/activate_voicemail:

Activate voicemail
==================

In order for a user to utilize voicemail in Axivox, the voicemail feature **must** be turned on in
the Axivox administrative portal. To begin using voicemail with a user, navigate to
`manage.axivox.com <axivox_admin_>`_. Then, log in with the appropriate administrator credentials.

On the left menu of the Axivox administrative panel, click into :guilabel:`Users`.

Then, click into the specific user the voicemail should be activated for. Under the section marked,
:guilabel:`Voicemail`, open the drop-down menu, and click on :guilabel:`Yes`.

Lastly, :guilabel:`Save` the change, then click :guilabel:`Apply changes` in the upper-right corner
of the screen.

Voicemail
=========

The next step is to set up the individual voicemail boxes on the Axivox administrative portal. To
access the portal, visit `manage.axivox.com <axivox_admin_>`_ and log in. Then, navigate to
:menuselection:`Voicemails`, located in the menu on the left.

If the voicemail option was activated in the user profile, using this process
:ref:`axivox/activate_voicemail`, then a voicemail is automatically created on the

> :guilabel:`Voicemails` pages.

Reviewer: ```suggestion
:guilabel:`Voicemails` page.
```

   :ref:`axivox/global_language`

Manually create voicemail
-------------------------

To manually create a new voicemail box, click :guilabel:`Add a voicemail` on the
:guilabel:`Voicemails` page. Or, edit an existing voicemail box, by clicking :guilabel:`Edit` to the
far-right of an existing voicemail box on the :guilabel:`Voicemails` page.

.. example::
   Suppose a sales or support team needs a general voicemail box. The voicemail would need to be
   created manually, and attached to an incoming number.

The new, manually-created voicemail box should be attached to an incoming number, so it can receive
messages. To do so, navigate to :menuselection:`Incoming numbers`, located in the menu on the left.
Then, click :guilabel:`Edit` to the far-right of the specific number the voicemail should be linked
to.

In the :guilabel:`Destination type for voice call` field, click the drop-down menu, and select
:guilabel:`Voicemail`. Then, open the drop-down menu on the next line labeled,
:guilabel:`Voicemail`, and select the manually-created voicemail box.

.. important::
   If an incoming number is capabable of receiving SMS/text messages, an additional field,

>    :guilabel:`Destination email address for Incoming SMS` is present.

Reviewer: ```suggestion
   :guilabel:`Destination email address for Incoming SMS`, is present.
```

   To determine whether an incoming number is capable of receiving SMS/text messages, click
   :guilabel:`Incoming numbers` from the menu on the left, then check the :guilabel:`SMS compatible`
   column for the incoming number.

Then, if applicable, in the field labeled, :guilabel:`Destination email address for Incoming SMS`,
enter an email to which incoming text messages sent to the incoming number can be received. Some
incoming numbers (US +1) in Axivox are capable of receiving text messages from individuals and
automated numbers.

Should this field be left empty, the default destination address is used, instead (as previously set
in the beginning of the process for manually creating a voicemail).

Once all desired configurations are complete, click :guilabel:`Save`, then click :guilabel:`Apply
changes` in the upper-right corner of the screen to implement the change into production.

Notifications
-------------

> Now, whenever a voicemail is received on any of the automatically pre-configured, or
> manually-linked, voicemail boxes, an email is sent to the user's email address, as listed in the

Reviewer: ```suggestion
Now, whenever a voicemail is received on any of the automatically pre-configured or
manually-linked voicemail boxes, an email is sent to the user's email address, as listed in the
```

:guilabel:`Voicemails` page, or in the user's Axivox profile.

This information can be accessed by navigating to :menuselection:`Users` in the left menu, and
clicking :guilabel:`Edit` next to the specific user in question.

.. _axivox/vm_forwarding:

Forwarding to voicemail
=======================

> In Axivox, there are also numerous forwarding settings for a user. To access these forwardings

Reviewer: ```suggestion
In Axivox, there are also numerous forwarding settings for a user. To access these forwarding
```

settings, go to `manage.axivox.com <axivox_admin_>`_ and log in.

Next, navigate to :menuselection:`Users`, located in the menu on the left.

From there, click into the specific user the forwarding should be added to. Then, open the
:guilabel:`Forwardings` tab.

If the user is busy on another call, or away from the phone, there is an option present in this tab
to :guilabel:`Send to voicemail as a last resort`, located in the :guilabel:`Forwarding on no
answer` and :guilabel:`Forwarding on busy` fields.

.. image:: vm_audio_messages/forwardings.png
   :align: center
   :alt: Send to voicemail as a last resort options highlighted on the Forwardings tab of the user.

If the :guilabel:`Send to voicemail as a last resort` box is ticked, when the forwarding actions
stated in each section are not successful, the caller is routed to the voicemail set on the
particular user.

.. seealso::

>    For more information on forwarding and transfers visit :ref:`axivox/forwardings_tab`.

Reviewer: ```suggestion
   For more information on forwarding and transfers, visit :ref:`axivox/forwardings_tab`.
```

:guilabel:`Apply changes` in the upper-right corner of the screen to implement the change.

.. _voip/audio_messages:

Audio messages
==============

It is possible to add audio messages *before* a customer's call is even taken, to inform them about
the waiting time for deliveries, the availability of a product, or any other important promotional
messages.

To record an audio message in Axivox, navigate to `manage.axivox.com <axivox_admin_>`_ and log in.

Next, click on :guilabel:`Audio messages` in the menu on the left. From the :guilabel:`Audio
messages` page, click :guilabel:`Add a message`.

Type in a :guilabel:`Name`, and click :guilabel:`Save`.

Upon clicking :guilabel:`Save`, the browser redirects back to the main :guilabel:`Audio messages`
page, where the newly-created message can be found on the list.

There are two different ways to make the audio message. The user could either record the message
over the phone, or type the message (in text), and select a computer-generated speaker to read the
message.

Record audio message
--------------------

To record an audio message over the phone, click the orange button labeled,

> :guilabel:`Record/Listen`, located to the right of the desired message on the list to record for, on
> the :guilabel:`Audio messages` page.

Reviewer: ```suggestion
:guilabel:`Record/Listen`, located to the right of the desired message on the list to record, on the
:guilabel:`Audio messages` page.
```

   this documentation: :doc:`axivox_config`.

Upon connecting to the Axivox audio recorder management line, a recorded French-speaking operator
provides the following options:

#. Press `1` to record a message.
#. Press `2` to listen to the current message.

Press either `1` or `2`, depending on whether or not there is already a message present in the
system for this particular audio message that requires a review, before recording a new one.

Record the new audio message after pressing `1`, then press `#` to end the recording.

The French-speaking operator returns to the line presenting the first set of questions again:

#. Press `1` to record a message.
#. Press `2` to listen to the current message.

Press `#` to end the call.

Write audio message
-------------------

To type the message, and select a computerized speaker to say the text, navigate to the
:menuselection:`Audio messages` in the menu on the left.

From the :guilabel:`Audio messages` page, select the blue button labeled, :guilabel:`Text message`,
next to the corresponding audio message :guilabel:`Name` that the message should be attached to.

> Doing so reveals :guilabel:`Convert text to message` pop-up window.

Reviewer: ```suggestion
Doing so reveals a :guilabel:`Convert text to message` pop-up window.
```

From the :guilabel:`Convert to text message` pop-up window, click the drop-down menu next to the
field labeled, :guilabel:`Voice`, and select an option for the :guilabel:`Text` to be read in.

After the :guilabel:`Voice` selection has been made, and the message has been written in the
:guilabel:`Text` field, click :guilabel:`Generate` to process the audio file.

The text is read in the same language it is written in the :guilabel:`Text` field. Should the

> language differ in the :guilabel:`Voice`, then an accent is used by the computerized speaker.

Reviewer: ```suggestion
language differ in the :guilabel:`Voice` field, then an accent is used by the computerized speaker.
```

Finally, when these steps are complete, click :guilabel:`Save` to save the audio message.

To implement the changes, click :guilabel:`Apply changes` in the upper-right corner of the screen.

.. image:: vm_audio_messages/sample-message.png
   :align: center
   :alt:  Convert text to message window with voice, text, generate button and save highlighted.

.. tip::

>    To set a greeting or audio message in a dial plan element double click on the element. This could
>    be a :guilabel:`Play a file` or even a :guilabel:`Menu` element in which the caller should
>    encounter an urgent message or a dial-by-number directory. For more information on  dial plans
>    see this documentation: :doc:`dial_plan_basics` or :doc:`dial_plan_advanced`.

Reviewer: ```suggestion
   To set a greeting or audio message in a dial plan element double-click on the element. This could
   be a :guilabel:`Play a file` element, or a :guilabel:`Menu` element, in which the caller should
   encounter an urgent message, or a dial-by-number directory.
   
   For more information on dial plans see this documentation: :doc:`dial_plan_basics` or
   :doc:`dial_plan_advanced`.
```

.. _axivox/music_on_hold:

Music on-hold
=============

Axivox has the option to add custom hold music to the call whenever a caller is waiting for their

> call to be answered. To add hold music on to the Axivox administrative portal, navigate to the

Reviewer: ```suggestion
call to be answered. To add hold music to the Axivox administrative portal, navigate to the
```

`manage.axivox.com <axivox_admin_>`_, and log in.

Then, click on :guilabel:`Music on hold` from the menu on the left, and a :guilabel:`Change the
music on hold` pop-up window appears.

> On the :menuselection:`Change the music on hold` pop-up window, click the :guilabel:`Choose File`
> button to select an MP3 (MPEG Audio Layer 3) or WAV (Waveform Audio File Format ) file to be
> uploaded.

Reviewer: ```suggestion
On the :guilabel:`Change the music on hold` pop-up window, click the :guilabel:`Choose File` button
to select an MP3 (MPEG Audio Layer 3) or WAV (Waveform Audio File Format ) file to be uploaded.
```
